documentation/src/main/asciidoc/ch09.asciidoc

```diff
@@ -77,15 +77,15 @@ case the specified element does not exist or is not constrained.
 
 The methods `getConstrainedProperties()`, `getConstrainedMethods()` and `getConstrainedConstructors()`
 return (potentially empty) sets with all constrained properties, methods and constructors,
-respectively. An element is considered constrained, if it has at least one constraint or is marked
+respectively. An element is considered constrained if it has at least one constraint or is marked
 for cascaded validation. When invoking `getConstrainedMethods()`, you can specify the type of the
 methods to be returned (getters, non-getters or both).
 
 [[section-property-descriptor]]
 === `PropertyDescriptor`
 
 The interface
-`link:$$https://docs.jboss.org/hibernate/beanvalidation/spec/2.0/api/index.html?javax/validation/metadata/PropertyDescriptor.html$$[PropertyDescriptor]` represents one given property of a
+https://docs.jboss.org/hibernate/beanvalidation/spec/2.0/api/index.html?javax/validation/metadata/PropertyDescriptor.html[`PropertyDescriptor`] represents one given property of a
```

Comment:
The links need to be checked manually anyway (typically, the anchors might have changed). But it will surely be easier to find them if we have a couple of URL to check.